src/DatumThese.ts

```diff
@@ -443,10 +456,9 @@ export const fromNullable = <E, A>(
       (e, a) => f(a, b)
     )(fa);
   
-/**
- * @since TODO
- */
- export const datumThese: Traversable2<URI> & TheseM1<D.URI> = {
+// TODO: After we bump the min bound to >= 2.10, replace this with individual helper fns
+// Also, fix the instance factories below to not re-use getMonad for better tree-shaking.
+const theseTDatum: Traversable2<URI> & TheseM1<D.URI> = {
```

Comment:
Since this module was never released, no sense in exporting the soon-to-be-deprecated mega-instance. The full functionality is exposed below with standalone instances.